.github/workflows/pylint.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install pylint
+        pip install -r requirements.txt
     - name: Analysing the code with pylint
       run: |
-        pylint $(git ls-files '*.py')
+        pylint $(git ls-files '*.py') --fail-under=7.0
```

Comment:
@kylos101 Added a flag to modify the behavior of pylint such that it will no longer give false negatives